src/services/s3.service.ts

```diff
@@ -0,0 +1,76 @@
+import * as aws from 'aws-sdk';
```

Comment:
Another open question, considering we are using typescript why we don't implement these kinds of services like Classes instead of just plain javascript const? Something like [this](https://github.com/rootstrap/nodejs-api-base/blob/master/src/services/sendgrid.service.ts) avoiding the loopback part

Reply:
Got it. Both approaches work for me, what does the rest say?

Reply:
It would be awesome to make this a TS class. Also, I think we can push even more the power of TS and have an Interface with needed behavior, that way in the future we can also have a set of uploaders (aws, firebease, gdrive, etc) and even a factory for creating those in runtime